curated_transformers/tokenization/bbpe_encoder.py

```diff
@@ -81,7 +83,12 @@ def byte_bpe_encoder_forward(model: Model, X: List[Doc], is_train: bool):
     return pieces, lambda dY: []
 
 
-def build_byte_bpe_encoder_loader_v1(*, vocab_path: Path, merges_path: Path):
+def build_byte_bpe_encoder_loader_v1(
+    *, vocab_path: Path, merges_path: Path
+) -> Callable[
+    [Model[List[Doc], List[Ragged]], Optional[List[Doc]], Any],
```

Comment:
I think it would be nice to have aliases for complex, repeatedly used types like `Model[List[Doc], List[Ragged]]`. Semantic descriptions are so much easies to read (e.g. this being something like `Tok2PiecesModel`.